nengo_loihi/neurons.py

```diff
@@ -8,6 +8,41 @@
 from nengo.params import NumberParam
 
 
+def loihi_lif_rates(neuron_type, x, gain, bias, dt):
+    # discretize tau_ref as per CxGroup.configure_lif
+    tau_ref = dt * np.round(neuron_type.tau_ref / dt)
+
+    j = neuron_type.current(x, gain, bias) - 1
+
+    out = np.zeros_like(j)
+    p = tau_ref + neuron_type.tau_rc * np.log1p(1. / j[j > 0])
+    out[j > 0] = (neuron_type.amplitude / dt) / np.ceil(p / dt)
+    return out
+
+
+def loihi_spikingrectifiedlinear_rates(neuron_type, x, gain, bias, dt):
+    j = neuron_type.current(x, gain, bias)
+
+    out = np.zeros_like(j)
+    p = 1. / j[j > 0]
+    out[j > 0] = (neuron_type.amplitude / dt) / np.ceil(p / dt)
+    return out
+
+
+def get_loihi_rates(neuron_type, x, gain, bias, dt):
```

Comment:
Could this be `loihi_rates`? I don't think the `get` adds to understanding what the function does.

Reply:
Sure